dataline-config/init/build.gradle

```diff
@@ -1,5 +1,5 @@
 // seed local directory with initial configurations.
 copy {
     from 'src/main/resources'
-    into '../data'
+    into '../../data'
```

Comment:
unfortunately gradle doesn't seem to have a great way of getting the root project dir so we don't have to worry about relative logic

Reply:
It actually does. Just updated the code